.github/workflows/acceptance.yml

```diff
@@ -214,10 +243,34 @@ jobs:
         uses: ./.github/actions/artifacts
         if: ${{ always() && needs.files-changed.outputs.backend_all == 'true' }}
 
+  backend-visual-diff:
+    if: always()
+    # This guarantees that we will only schedule Visual Snapshots if all
+    # workflows that generate artifacts succeed
+    needs: [backend]
+    name: trigger backend visual snapshot
+    runs-on: ubuntu-20.04
+    timeout-minutes: 20
+    steps:
```

Comment:
I think these `-visual-diff` jobs can be tacked onto the end of their parent job directly -- then the failure check can be removed?

Reply:
Yeah I'm realizing the `triggers visual snapshot` was a required check as well as `Visual Snapshot`, which may explain this structure which previously used -visual-diff to pass or fail the `triggers visual snapshot` check.

Current proposal is to change required checks to "<Frontend|Backend|Chartcuterie> Visual Snapshot", remove the required trigger check and rely on fake "<*> Visual Snapshot" to handle the case where snapshots are not run.

The only risk here would be a snapshot runs and fails, which will leave the required `* Visual Snapshot` check as a pending workflow that will never start or fail.

Reply:
oh right, because that doubled as the "collector" before hmmm

ugh that means we need to split this into even more steps T.T